QuizApp/Scenes/LoginPage/ViewController/LoginViewController.swift

```diff
@@ -9,15 +9,143 @@ import UIKit
+    private let blueBackground: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.image = .blueBackground
+        return imageView
+    }()
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "ჩემი პირველი ქვიზი"
+        label.font = .systemFont(ofSize: FontSizes.xl, weight: .black)
+        label.textColor = .white
+        return label
+    }()
+    
+    private let illustration: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.image = .loginPageIllustration
+        return imageView
+    }()
+    
+    private let loginStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.spacing = Sizing.stackViewSpacing
+        stackView.alignment = .center
+        return stackView
+    }()
+    
+    private let input: UITextField = {
+        let input = UITextField()
+        input.isUserInteractionEnabled = true
+        input.textColor = .neutralGrey
+        input.textAlignment = .center
+        let placeholderText = "შეიყვანეთ სახელი"
+        let attributes: [NSAttributedString.Key: Any] = [
+            .font: UIFont.systemFont(ofSize: FontSizes.xs)
+        ]
+        input.attributedPlaceholder = NSAttributedString(string: placeholderText, attributes: attributes)
+        input.layer.borderWidth = Sizing.inputBorder
+        input.layer.borderColor = Colors.border.cgColor
+        input.layer.cornerRadius = Sizing.inputRadius
+        return input
+    }()
     
+    private let startButton = YellowRoundedButton(
+        title: "ქვიზის დაწყება",
+        width: Sizing.startButtonWidth,
+        height: Sizing.startButtonHeight,
+        radius: Sizing.startButtoRadius,
+        fontSize: FontSizes.xs
+    )
     
+    // MARK: - Constraints
+    private var blueBackgroundTopConstraint: NSLayoutConstraint!
+    
+    // MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .blue
+        setupUI()
+        setupKeyboardObservers()
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        removeKeyboardObservers()
+    }
+    
+    // MARK: - UI Setup
+    private func setupUI() {
+        setupView()
+        setupViewHierarchy()
+        setConstraints()
+    }
+    
+    private func setupView() {
+        view.backgroundColor = .white
+    }
+    
+    private func setupViewHierarchy() {
+        loginStackView.addArrangedSubviews(input, startButton)
+        view.addSubviews(blueBackground, titleLabel, illustration, loginStackView)
+        view.bringSubviewToFront(titleLabel)
+    }
+    
+    private func setConstraints() {
```

Comment:
Since this function is too big, you can have seperate setUpConstraints functions for background fro title and they will be called inside this function. it will be cleaner I guess. you can try it